### PR TITLE
add sample for follow parameter of copy.py

### PR DESCRIPTION
**Pull request by ptux**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

copy.py

### CODE REVIEW

**Comment by ptux** [comment]
ready_for_review

**Comment by ptux** [comment]
bot_status

**Comment by ptux** [comment]
ship_it

**Comment by ptux** [comment]
@abadger 

reminder.

File: lib/ansible/modules/files/copy.py [hunk 1]
```diff
@@ -164,6 +164,18 @@
...
+  copy:
+    src: /etc/foo.conf
+    dest: /path/to/link # /path/to/link is link to /path/to/file
+    follow: True
+
+- name: if follow is False, /path/to/link will become a file and be overwritten by contents of foo.conf
+  copy:
+    src: /etc/foo.conf
+    dest: /path/to/a/link # /path/to/link is link to /path/to/file
+    follow: False
```
Comment (abadger) [inline lib/ansible/modules/files/copy.py:178]: There is an extra /a/ in this sample path.

**Review by abadger — COMMENTED** [review]
This looks good one the one correction is made

**Comment by abadger** [comment]
Thanks @ptux!

**Pull request merged**